library/Kubernetes/Web/Conditions.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/* Icinga for Kubernetes Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Kubernetes\Web;
+
+use Icinga\Module\Icingadb\Util\PluginOutput;
+use Icinga\Module\Icingadb\Widget\PluginOutputContainer;
+use ipl\Html\Attributes;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\I18n\Translation;
+use ipl\Web\Widget\Icon;
+use ipl\Web\Widget\TimeAgo;
+
+abstract class Conditions extends HtmlDocument
+{
+    use Translation;
+
+    abstract protected function getConditions();
+
+    abstract protected function getVisual($status, $type): array;
+
+    public function assemble()
+    {
+        $conditions = $this->getConditions();
+        if (empty($conditions)) {
+            return;
+        }
+
+        $listItems = [];
+
+        foreach ($conditions as $condition) {
+            $pluginOutput = $condition->reason;
+            $pluginOutput .= $condition->message ? ": " . $condition->message : '';
```

Comment:
You mix double and single quotes here and there. Please use double quotes only where necessary.